css/style.css

```diff
@@ -1 +1,1087 @@
-body {}
\ No newline at end of file
+:root {
+  --btn-primary-color: rgba(33, 231, 134, 1);
+  --secondary-color: rgba(20, 27, 34, 1);
+  --article-bg-color: rgba(255, 255, 255, 0.05);
+  --main-background-color: rgba(13, 13, 17, 1);
+  --text-color: rgba(194, 195, 197, 1);
+  --inactive-color: rgba(136, 139, 142, 1);
+  --text-font: "Poppins", sans-serif;
+  --btn-header-font: "Bakbak One", sans-serif;
+}
+
+/********************** General css **********************************/
+
+* {
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  background-color: rgba(4, 11, 17, 1);
+  font-family: "Bakbak One", sans-serif, "Roboto", Arial, sans-serif;
```

Comment:
Use here `--btn-header-font` variable. 
`"Roboto", Arial, sans-serif` is just example. In your case it is: 
font-family: "Bakbak One", Arial, sans-serif;

https://crystallize.com/blog/web-safe-fonts#:~:text=System%20Fonts%3A%20System%20fonts%20are,consistent%20rendering%20on%20different%20devices.

Reply:
#fixed